examples/stargazers/webpack.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
 
   module: {
     loaders: [
-      { test: /\.js$/, exclude: /node_modules|dist/, loader: "babel-loader" },
+      { test: /\.js$/, exclude: /node_modules|dist/, loader: "babel-loader?stage=0" },
```

Comment:
Stage 0 is so static class properties work which are required so that the class has a display name.